src/main/groovy/io/seqera/wave/service/persistence/impl/SurrealPersistenceService.groovy

```diff
@@ -95,6 +95,7 @@ class SurrealPersistenceService implements PersistenceService {
 
     @Override
     void saveBuild(WaveBuildRecord build) {
+        build.condaFile = truncateLargeCondaFile(build.condaFile)
```

Comment:
Think can also happen for the docker file in principle

Reply:
ok will add it too
I think when condafile is present continerfile will be our template so it wont exceed size, but when we just have dockerfile, size can be bigger

Reply:
done